tangelo/linq/translator/translate_cirq.py

```diff
@@ -164,3 +165,45 @@ def translate_c_to_cirq(source_circuit, noise_model=None, save_measurements=Fals
                     target_circuit.append(depo(*depo_list))  # gates targeted
 
     return target_circuit
+
+
+def translate_op_to_cirq(qubit_operator):
+    """Helper function to translate a Tangelo QubitOperator to a cirq linear
+    combination of Pauli strings.
+
+    Args:
+        qubit_operator (tangelo.toolboxes.operators.QubitOperator): Self-explanatory.
+
+    Returns:
+        (cirq.ops.linear_combinations.PauliSum): Cirq summation of Pauli strings.
+    """
+    from openfermion.transforms import qubit_operator_to_pauli_sum
+
+    return qubit_operator_to_pauli_sum(qubit_operator)
+
+
+def translate_op_from_cirq(qubit_operator):
+    """Helper function to translate a cirq linear combination of Pauli strings
+    to a Tangelo QubitOperator.
+
+    Args:
+        qubit_operator (cirq.ops.linear_combinations.PauliSum): Self-explanatory.
+
+    Returns:
+        (tangelo.toolboxes.operators.QubitOperator): Tangelo qubit operator.
+    """
+    import cirq
+
+    tangelo_op = QubitOperator()
```

Comment:
I think this line can be discarded if L207 reads `=` instead of `+=` ?

Reply:
I think it would work with only one Pauli word, but not with a summation of Pauli words.